### DISCUSSION TITLE
Remote console shows no data after serialbattery installation

### DISCUSSION THREAD

**Discussion by gsxarne** [Louisvdw/dbus-serialbattery#342] [Q&A] [answered]
Hi all, i probably made the mistake to install the latest serialbattery Version onto my Cerbo GX with Firmware Version V2.75. The FaQ  says,  that 2,8 is  required.  Anyhow. Now my remote console doesnt show any content and also doesn't react on pressing the buttons:
![image](https://user-images.githubusercontent.com/6322427/206874154-6b7947aa-7bf0-4ddc-afbd-6a337366b8aa.png)

Luckily i am still able to connect via Telnet, and so i tried to run disabledriver.sh , but still the remote console is empty.

Has someone a  clue how i could solve this issue?

**Comment by gsxarne** [comment]
i found this one here and tried it . But it  doesn't work

https://github.com/Louisvdw/dbus-serialbattery/issues/325

Is there a command which deinstalls the complete package?

**Reply by Louisvdw** [reply, marked as answer]
The white screen is when the GUI changes does not match the venusOS version. Everything will still work but you cannot change settings or see anything. That command in #325 will revert to the original GUI file and should solve it. (But without the cell voltage display)
Once you have the display working then upgrade to VenusOS 2.91 or 2.92 and your can install it again to get the cell voltage display, or just keep it without that.

**Reply by gsxarne** [reply]
Hi, i copied 
`cp -f /opt/victronenergy/gui/qml/PageBattery.qml.backup /opt/victronenergy/gui/qml/PageBattery.qml`

Into a Putty console while logged in as a superuser and did a reboot aftwerwards, But as i said it doesn't work neither.

**Reply by Louisvdw** [reply]
You can do command line firmware updates for the GX device in SSH by running this command and following the prompts
> /opt/victronenergy/swupdate-scripts/check-updates.sh

This will also reset the driver, but if you have not run the disable command the driver will reinstall itself again, but this time it should work with the gui changes.

**Reply by gsxarne** [reply]
Thank you very much, this hint with the software updated saved my cerbo. I'll give you a donation via Paypal, and if you are in the norhtern Parts of Germany (or southern Part of Denmark) i'll invite you for a beer or two.

**Reply by gsxarne** [reply]
Hmm okay, I still have the issue that the Values from the Energymeter are not shown. 
I guess i have to do a factory reset

**Reply by Louisvdw** [reply]
Can you give more details. Which BMS are you using and is it not picked up at all or is it only some value not displayed?
You can also work through https://github.com/Louisvdw/dbus-serialbattery/wiki/Troubleshoot and the FAQ. If that does not help could you send a screenshot and the serialbattery log file.

**Comment by entropia1ac** [comment]
update Cerbo GX to v.2.92 via SSH. Description on Victron developer website

**Reply by gsxarne** [reply]
Can you help with a link?

**Comment by entropia1ac** [comment]
I had this problem quite a long time ago. You need to look on the Victron beta testers site

**Comment by cyborgxxl** [comment]
Hmm unfortunately I can't get rid of the white screen. I use the firmware 2.94 and since the update I have a white screen in the remote console. The problem is that SSH isn't active and so I can't connect. Is it possible to reset the nanopi image to the factory state somehow?

![photo_2023-05-08_15-10-38](https://user-images.githubusercontent.com/6112343/236833631-e8943b9d-88a4-40c1-a99a-fee8851eaecb.jpg)
![photo_2023-05-08_15-10-43](https://user-images.githubusercontent.com/6112343/236833650-f699b0b0-a782-4638-9193-78134cc0f4c4.jpg)

**Reply by mr-manuel** [reply]
1. What did you update?
2. Which version of the driver have you installed?
3. Have you installed also other drivers/packets like GUI mods?
4. Do you have physical access to the device?

**Reply by cyborgxxl** [reply]
1. I installed the driver "v1.0.0beta20230505a"
2. I updated from 2.93 to 2.94
3. Nope, I have no mods.
4. Yes, I have physical access to the device.

**Reply by mr-manuel** [reply]
Get a USB drive and format it with FAT32. Copy [venus-data.RESTORE_GUI.tar.gz](https://github.com/Louisvdw/dbus-serialbattery/files/11421622/venus-data.RESTORE_GUI.tar.gz) to the USB drive and rename it to `venus-data.tar.gz`. Plug the USB drive into the device and reboot/powercycle it. Wait a bit and check, if you see again something.

If this don't work or you only see your GUI for a moment I have to change the script.

**Reply by mr-manuel** [reply]
How you installed it, without SSH access?

**Reply by cyborgxxl** [reply]
Thank you very much! If I put the venus-data.tar.gz on the USB stick and the powercycle device, the screen on the device stops at "Victron Energy". Apparently, the device then no longer boots and the Victron hotspot is also not displayed to me.

I had SSH with version 2.93, after the update to 2.94 it no longer accepts my root password.

**Reply by mr-manuel** [reply]
It takes 60 seconds, that the script starts, because it waits, that all other scripts already run (like `rc.local`). The root password is reset with every firmware update.

**Reply by cyborgxxl** [reply]
Ok, I will test it again.

**Reply by mr-manuel** [reply]
Else you can also put this `venus-data.tar.gz` on the USB. https://github.com/mr-manuel/venus-os_dbus-serialbattery/releases/tag/v1.0.0-beta_20230508
Today I reworked the install procedure via USB and it should fix your problem persistently.

**Reply by cyborgxxl** [reply]
Ok, thank you! Your script worked perfect after 60 seconds. I guess I'm a little impatient. ;-)

I then tried to install the driver via "wget -O /tmp/install.sh https://raw.githubusercontent.com/Louisvdw/dbus-serialbattery/master/etc/dbus-serialbattery/install.sh" but I always get an error 404. With the link you provided with the revised version, it worked without any problems.

**Reply by mr-manuel** [reply]
You were to fast :) we merged the PR right now. So now it works
